### PR TITLE
Feature/pending

### PR DESCRIPTION
**Pull request by rickmansfield**
## Description

Updates HomeContainer to render PendingAproval.js which now contains code for a sidebar and a message regarding the user's status. Here the goal was to ensure a new user that was still "pending" before being approved/denied a mentor/mentee role could see their application status. 

Fixes #
Card 7: As a pending user, after I register, I need to see a pending application message along with other pertinent information on my status.

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

https://www.loom.com/share/54c7164d3fdb4a19b1360b69ea468daa

at the end of this video, I was cut off. I simply ended by explaining that we removed the unnecessary routing from src>index.js for pendingApprovals

### CODE REVIEW

**Review by Retrofitt — APPROVED** [review]
Loom video does a great job of explaining the code you implemented. 
The code itself looks good, easy to read, and clean. Excellent work.

File: src/index.js [hunk 1]
```diff
@@ -64,7 +63,6 @@ function App() {
         <Route path="/menteeapplication" component={Mentee} />
         <Route path="/mentorapplication" component={Mentor} />
         <Route path="/login" component={LoginPage} />
-        <Route path="/pendingapproval" component={PendingApproval} />
         <Route path="/availability" component={Availability} />
```
Comment (ayv8er) [inline src/index.js:66]: Would we be placing these routes, including this new pending approval in a Secure Route?

**Review by ayv8er — APPROVED** [review]

**Review by cyberkade — CHANGES_REQUESTED** [review]
No hate against color customized bars, but I remember you saying you were gonna take this out before finishing the PR. Did you finish up and commit everything you wanted to?

**Review by teresafranxman97 — APPROVED** [review]
Very lucky to have been able to see this feature in action! Code is clean and readable, I also like the fact that pending users are still able to see the dark mode feature on their dashboard! Adds a great user experience to preview one of many great features the app has to show! Awesome job!

**Comment by RaphaelHebert** [comment]
Neat code. Pull request helps to follow your work and review the code. Well done, great job.

**Review by cyberkade — APPROVED** [review]
Let's get this merged!

**Pull request merged**